scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -178,18 +211,7 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
      */
     private fun scanPackage(scannerDetails: ScannerDetails, pkg: Package, outputDirectory: File,
                     downloadDirectory: File): List<ScanResult> {
-        val scanResultsForPackageDirectory = File(outputDirectory, pkg.id.toPath()).apply { safeMkdirs() }
-        val resultsFile = File(scanResultsForPackageDirectory, "scan-results_${scannerDetails.name}.$resultFileExt")
-
-        val storedResults = ScanResultsStorage.read(pkg, scannerDetails)
-
-        if (storedResults.results.isNotEmpty()) {
-            // Some external tools rely on the raw results filer to be written to the scan results directory, so write
-            // the first stored result to resultsFile. This feature will be removed when the reporter tool becomes
-            // available.
-            resultsFile.mapper().writeValue(resultsFile, storedResults.results.first().rawResult)
-            return storedResults.results
-        }
+        val resultsFile = getResultsFile(scannerDetails, pkg, outputDirectory)
```

Comment:
The docs (GitHub doesn't let me comment on line 207) now need to be adjusted as this now really only scans, but does not read from the storage anymore. Maybe you can move the outdated docs to `readFromStorage()` with some minor modifications?

Reply:
Done.